src/Estimators/include/BipedalLocomotion/RobotDynamicsEstimator/SubModelKinDynWrapper.h

```diff
@@ -81,7 +85,12 @@ class SubModelKinDynWrapper
      * updateDynamicsVariableState updates the value of all the member variables containing
      * information about the robot kinematics and dynamics
      */
-    bool updateDynamicsVariableState();
+    bool updateDynamicsVariableState(bool updateRobotDynamicsOnly);
```

Comment:
Usually boolean argument is not really readable.
Imagine someone reading your code for the first time
```cpp
updateDynamicsVariableState(false);
```
can him/her guess what that ``false`` does?

Usually it is more convenient to use an enum class. Like
```cpp
enum class UpdateMode {
    Full,
    RobotDynamicsOnly
}
```
In this way, the code will become
```cpp
updateDynamicsVariableState(UpdateMode::Full);
```
that is more readable, and will be also easily extendible.

In this case, the method is protected so it should not matter a lot, but it is something you may want to consider.